src/tests/testSetup.ts

```diff
@@ -0,0 +1,60 @@
+import { runMutations } from "automutate";
+import fs from "node:fs";
+import path from "node:path";
+import ts from "typescript";
+
+import { fillOutRawOptions } from "../options/fillOutRawOptions.js";
+import { RawTypeStatOptions } from "../options/types.js";
+import { createTypeStatProvider } from "../runtime/createTypeStatProvider.js";
+
+export const runMutationForTest = async (
+	dirPath: string,
+	originalFileName = "original.ts",
```

Comment:
[Refactor] To simplify things a bit for consumers, how about making this non-configurable?

```ts
	const originalFileName = (await fs.readdir(dirPath)).find((file) =>
		file.startsWith("original."),
	);
	if (!originalFileName) {
		throw new Error(`${dirPath} should have a file named original.*`);
	}
```

Reply:
Yeah I think the limited scale of these tests makes me lean towards thinking hiding details is ok for now. The only difference between tests that we're hiding is the file extension for `original.*`. If it gets more complicated then yeah, I think for clarity's sake it'd be better to lean back towards being explicit.